src/components/Compare/HoursTable.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslation, TFunction } from "react-i18next";
+
+import {
+  CareProvider,
+  DayOfWeek,
+  DAYS_OF_THE_WEEK,
+  WeeklyHours,
+} from "../../types";
+import CompareStripedRows from "./CompareStripedRows";
+
+const formatHoursDisplay = (
+  hours: WeeklyHours,
+  day: DayOfWeek,
+  t: TFunction
+): string => {
+  if (!hours) {
+    return "--";
```

Comment:
question for @admoorgit  - what should we display if one provider has no hours data? in those cases, we display a "please contact provider" msg in the detail section, but that's quite verbose and doesn't rly make sense in each table cell here

also - should we display something diff (e.g. no table and just a message) if both providers are missing hours data?